### ISSUE TITLE
Icon request: fa-artstation and fa-artstation-square

### ISSUE THREAD

**Issue by elantzb** [FortAwesome/Font-Awesome#5354]
I've seen people using Font Awesome for their social networking icons, but they all have to hack in their own icons for Artstation.

The website is here:
http://www.artstation.com/

The basic icon is here:
![image](https://cloud.githubusercontent.com/assets/4442027/5657112/c7801254-969d-11e4-81f9-f5bc43fced35.png)

The square icon is here:
![image](https://cloud.githubusercontent.com/assets/4442027/5657115/d795b02c-969d-11e4-9249-8db83146386a.png)

Thanks much!

**Comment by miskopo** [comment]
I would like to have this icon too, please

**Comment by Fnitrox** [comment]
+1 for this one

**Comment by enzoz** [comment]
Would be awesome to have this icon on font-awesome :+1:

**Comment by ianchase09** [comment]
has there been any success with this ? def. want this

**Comment by krystalcampioni** [comment]
+1

**Comment by illustribe** [comment]
+1! :+1:

**Comment by Sythenz** [comment]
+1!

**Comment by saphiresouldier** [comment]
big +1 from me, Artstation is getting more important as an portfolio site for artists everyday and it would be awesome to have the icon included!

**Comment by jbltx** [comment]
+1

**Comment by redoper** [comment]
+1

**Comment by jbltx** [comment]
For people who can't wait, you can find m'y custom font which use artstation and Polycount icons at my website http://www.mickael-b.com

**Comment by SirMoptop** [comment]
+1

**Comment by nervous-entity** [comment]
+1

**Comment by JFDesigner** [comment]
:+1:

**Comment by leonardteo** [comment]
Hi all,

The official icon vector files are available here:
https://www.artstation.com/about/logo

Leo

**Comment by l0stiwisd0m** [comment]
+1

**Comment by serhiikorol** [comment]
+1

**Comment by gvalmon** [comment]
+1

**Comment by ondrejholan** [comment]
+1

**Comment by TylerSangster** [comment]
+1

**Comment by mcwright** [comment]
+1

**Comment by jdugarte** [comment]
+1

**Comment by ntzog** [comment]
+1

**Comment by sangster** [comment]
+1

**Comment by djonatanb** [comment]
+1

**Comment by dansolo** [comment]
+!

**Comment by clarencemartens** [comment]
+1

**Comment by xsynthz** [comment]
+1

**Comment by moth3r** [comment]
+1

**Comment by Sullum97** [comment]
+1

**Comment by ajabotomey** [comment]
+1

**Comment by opico** [comment]
+1

**Comment by kevin1153** [comment]
+1

**Comment by Nazarsokol** [comment]
+1

**Comment by jbltx** [comment]
Some people asked me how to use my custom icons so while we are waiting for this in FontAwesome, here's mine.

1 - Download these files : 
http://www.mickael-b.com/assets/fonts/cg-icons.svg
http://www.mickael-b.com/assets/fonts/cg-icons.eot
http://www.mickael-b.com/assets/fonts/cg-icons.ttf
http://www.mickael-b.com/assets/fonts/cg-icons.woff

2 - Put these files in `/assets/fonts/` for example (like my website).

3 - Add this line in your main `.css` file : 

```css
@font-face{
  font-family:cg-icons;
  src:url(/assets/fonts/cg-icons.eot?z5qas7);
  src:url(/assets/fonts/cg-icons.eot?z5qas7#iefix) format('embedded-opentype'),url(/assets/fonts/cg-icons.ttf?z5qas7) format('truetype'),url(/assets/fonts/cg-icons.woff?z5qas7) format('woff'),url(/assets/fonts/cg-icons.svg?z5qas7#cg-icons) format('svg');
  font-weight:400;
  font-style:normal;
}
i.cg{
  font-family:cg-icons!important;
  speak:none;
  font-style:normal;
  font-weight:400;
  font-variant:normal;
  text-transform:none;
  line-height:1;
  -webkit-font-smoothing:antialiased;
  -moz-osx-font-smoothing:grayscale;
}
.cg-artstation:before{
  content:"\e900";
}
.cg-polycount:before{
  content:"\e901";
}
```

4 - Use artstation and polycount icons using `<i class="cg cg-artstation">` and `<i class="cg cg-polycount">`.

**Comment by ea-bruno** [comment]
+1

**Comment by RafaelDeJongh** [comment]
+1

**Comment by geothefaust** [comment]
+1

**Comment by pasanghyolmo** [comment]
+1

**Comment by tweikiang** [comment]
+1

**Comment by Sythenz** [comment]
Devs, an approximate of the required '+1' amount that it takes to push this as a request would be appreciated.

**Comment by LizzleCoop** [comment]
+1

**Comment by TheShadyColombian** [comment]
+1

**Comment by GMoney2** [comment]
+1 ...Attached a couple icons i created too. Feel free to use them as needed. Thanks.
![art_station_color](https://cloud.githubusercontent.com/assets/6051964/18768680/fa22f7e2-80f4-11e6-8a74-6cb10912d768.png)
![art_station_transparent](https://cloud.githubusercontent.com/assets/6051964/18768688/000703ec-80f5-11e6-80a7-f3efec5ecf16.png)

**Comment by davidlyons** [comment]
+1

**Comment by leonardteo** [comment]
@davegandy Just wondering, how much interest would we need to have the ArtStation icon added to the FontAwesome library? It's already ranked at 2000 on Alexa, making it one of the largest sites in the world and comparable to many of the social networks on FA. 
The official vector files are available at https://www.artstation.com/about/logo

Thanks!

**Comment by GMoney2** [comment]
Agreed. I love the Font Awesome library but i think we've shown there's legitimate, serious interest here so if you could let us know what's needed to get this done, I'm sure we can help. Thanks.

**Comment by raymestalez** [comment]
+1

**Comment by wulfgarrr** [comment]
+1

**Comment by kriskropd** [comment]
+1

Wait - why isn't this included yet? The logo vector is there and everything.

**Comment by joewestcott** [comment]
+1

**Comment by tiffz** [comment]
+1

**Comment by LikeMyBread** [comment]
+1

**Comment by MarkBTomlinson** [comment]
This has been on here for over two years and no response on when it is going to be added. As Pinterest has 3 icons at least it seems a small request to ask for one for the best CG site on the web.
1+

**Comment by donavanneil** [comment]
Yeah, this is a really obvious one. It would be really nice to see this added!

**Comment by elonliu** [comment]
+1 for this!

**Comment by LeoRaposo** [comment]
+1

**Comment by rnakanishi** [comment]
+1

**Comment by Sythenz** [comment]
Two years, decided instead to create my own Font library for CG related platforms. Will post here when it's ready - will open source it.

**Comment by php-require** [comment]
+1

**Comment by excalith** [comment]
+1

**Comment by Hawkbat** [comment]
+1

**Comment by ErisAnn** [comment]
+1

**Comment by Mordil** [comment]
+1

**Comment by damirgmartin** [comment]
Come on, give us the icon, pleeease.

**Comment by moth3r** [comment]
Why this request is still ignored considering Artstation is in the 100 most visited internet websites. That was even officially requested so please add the icon. Thank you!

**Comment by Alexvanderlinde** [comment]
+1

**Comment by snaik27** [comment]
+1

**Comment by myslibora** [comment]
+1

**Comment by tribute2nujabes** [comment]
+1

**Comment by diana-wang** [comment]
+1

**Comment by gorenburg** [comment]
+1. it's time add it to v5!

**Comment by ntzog** [comment]
+1

**Comment by fiftywalrus** [comment]
+1

**Comment by pedro-couto** [comment]
+1

**Comment by Jens-roesing** [comment]
+1

**Comment by ctaminian** [comment]
Can't believe this isn't there yet! +1

**Comment by smeerws** [comment]
+1

**Comment by mohsenkhanpour** [comment]
+1

**Comment by stobenski** [comment]
+1

**Comment by Niclas3er** [comment]
+1000 Please create that icon, I really need it!

**Comment by quasiagent1** [comment]
+1 Three years and counting for this? Has anyone come up with a work around? This has become a very important site for a lot of us...

**Comment by ianchase09** [comment]
so many other threads closed because it has been ignored for 3 years and is arguably the leading portfolio website for many artists. FA whats the deal?

**Comment by SoloSalsero** [comment]
I need this!

**Comment by talbs** [comment]
Thanks for your patience, gang. This was released in our latest version, 5.6.0 - https://fontawesome.com/icons/artstation.

**Comment by geothefaust** [comment]
> Thanks for your patience, gang. This was released in our latest version, 5.6.0 - https://fontawesome.com/icons/artstation.

And the people rejoiced!

Thank you!

**Comment by RafaelDeJongh** [comment]
Sweet!
